packages/python/readme_metrics/tests/PayloadBuilder_test.py

```diff
@@ -2,6 +2,8 @@
 import requests
 import json
 
+from .fixtures import Environ
```

Comment:
These changes supposed to be in this PR?

Reply:
I'll close this PR, I believe it was just a test, but I forget for what now

Reply:
Oh it was probably to test the python job, based on the branch name `test_python_github_job`